src/link.d

```diff
@@ -806,45 +807,49 @@ version (Windows)
         }
         // Normalize executable path separators, see Bugzilla 9330
         cmd = toWinPath(cmd);
-        version (CRuntime_Microsoft)
+        // Open scope so dmd doesn't complain about alloca + exception handling
         {
-            // Open scope so dmd doesn't complain about alloca + exception handling
+            // Use process spawning through the WinAPI to avoid issues with executearg0 and spawnlp
+            OutBuffer cmdbuf;
+            cmdbuf.writestring("\"");
+            // write cmd without any quotes
             {
-                // Use process spawning through the WinAPI to avoid issues with executearg0 and spawnlp
-                OutBuffer cmdbuf;
-                cmdbuf.writestring("\"");
-                cmdbuf.writestring(cmd);
-                cmdbuf.writestring("\" ");
-                cmdbuf.writestring(args);
-
-                STARTUPINFOA startInf;
-                startInf.dwFlags = STARTF_USESTDHANDLES;
-                startInf.hStdInput = GetStdHandle(STD_INPUT_HANDLE);
-                startInf.hStdOutput = GetStdHandle(STD_OUTPUT_HANDLE);
-                startInf.hStdError = GetStdHandle(STD_ERROR_HANDLE);
-                PROCESS_INFORMATION procInf;
-
-                BOOL b = CreateProcessA(null, cmdbuf.peekString(), null, null, 1, NORMAL_PRIORITY_CLASS, null, null, &startInf, &procInf);
-                if (b)
+                const(char)* start = cmd;
```

Comment:
Some description of what the transformation is supposed to be would be nice, so it could be compared with the code.